README.md

```diff
@@ -1,118 +1,47 @@
-- Click "Create repository from template"
-
-<img src="./docs/imgs/create_from_template.svg" width="50%"/>
-
-### Setup branch protections
-
-Once the repository has been created, the CI stored in this template (`.github/workflows`) will run. Ensure initial CI completes successfully before proceeding.
-
-<img src="./docs/imgs/verify_ci.svg" width="50%"/>
-
-- In Settings > Branches:
-    - Click "Add branch protection rule"
-    - Enter "main" for the Branch name pattern
-    - Check the following:
-        - Require a pull request before merging
-        - Require approvals
-            - Minimum of 1
-        - Dismiss stale pull request approvals when new commits are pushed
-    - Require status checks to pass before merging
-    - Require branches to be up to date before merging
-    - In the search box add the following:
-        - pass/fail checks
-        - Analyze (javascript)
-
-<img src="./docs/imgs/add_branch_protection_rule.svg" width="50%"/>
-
-<img src="./docs/imgs/the_rules.svg" width="50%"/>
-
-### Setup other Github settings
-- In Settings > General:
-  - Under "Features": 
-    - Enable/disable features that you want for your project. For example, turn off the Wiki if your project won't be using it
-  - Under "Pull Requests": 
-    - Enable only the merge options your project should support
-    - Check "Always suggest updating pull request branches" to encourage pull requests to be updated when they deviate from `main`
-    - Check "Automatically delete head branches" to keep set Github to automatically delete branches once they are merged into `main` in a PR
-
-<img src="./docs/imgs/pull_request_settings.svg" width="50%"/>
-
-- In Settings > Collaborators:
-  -  Add all collaborators that should have access to the git repo
-
-
-- In Settings > Code security and analysis:
-  - Click "Enable" for "Dependabot alerts"
-  - Click "Enable" for "Dependabot security updates"
-  - The other security features should already be enabled:
-    - "Dependabot version updates" is controlled by `.github/dependabot.yml`
-    - "Code scanning" is controlled by `.github/codeql-analysis.yml`
-    - "Secret scanning" is enabled by default
-
-<img src="./docs/imgs/code_sec_analysis.svg" width="50%"/>
-
-### Clone the repo to your local development environment
-
-Deploy the infrastructure from the infra folder by following the README.md instructions... I need to create the CD from [WMDP-96 Setup github actions for CD](https://wicmtdp.atlassian.net/browse/WMDP-96) then redo the steps above to verify if cd.yml will run, should also consider renaming ci.yml
+1. [Set up team workflow](./template-only-docs/set-up-team-workflow.md)
+2. [Set up resources needed to manage infrastructure as code (IaC)](./template-only-docs/set-up-infrastructure-as-code.md)
+3. [Set up continuous integration](./template-only-docs/set-up-ci.md)
+4. [Set up application](./docs/infra/set-up-app.md)
+5. [Set up application environments](./docs/infra/set-up-app-env.md)
```

Comment:
I like this index of template-only-docs — might be nice for the docs README (or the docs/infra/README.md) to have an index of pages since the pages themselves are not ordered in any way.

(Non-blocking; just a thought for the future.)

Reply:
Sounds good, added `docs/infra/README.md`